open_seq2seq/parts/cnns/conv_blocks.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2018 NVIDIA Corporation
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+from six.moves import range
+
+import tensorflow as tf
+
+
+def conv_actv(type, name, inputs, filters, kernel_size, activation_fn, strides,
```

Comment:
Why do we need this function? If BN is not done, then it's possible to just specify activation as an argument to the native tf.layers.conv1d/conv2d and there is no need to have a separate helper function for that.

Reply:
If we have this function its easier to make code generic in the encoder.